packages/flutter/lib/src/material/page.dart

```diff
@@ -91,6 +85,12 @@ class MaterialPageRoute<T> extends PageRoute<T> {
   @override
   Color get barrierColor => null;
 
+  /// Whether this route is a full-screen dialog.
+  ///
+  /// Prevents [startPopGesture] from poping the route using an edge swipe on
+  /// iOS.
+  bool fullscreenDialog;
```

Comment:
Looks like this got moved down. There's sometimes reasons to break this convention, but generally, the fields should follow the constructor and be in the same order as the constructor arguments.